src/KOKKOS/pair_dpd_ext_kokkos.cpp

```diff
@@ -41,8 +41,8 @@ using namespace LAMMPS_NS;
 
 
 template<class DeviceType>
-PairDPDExtKokkos<DeviceType>::PairDPDExtKokkos(class LAMMPS *lmp) :
-  PairDPDExt(lmp) ,
+PairDPDExtKokkos<DeviceType>::PairDPDExtKokkos(class LAMMPS *_lmp) :
+  PairDPDExt(_lmp) ,
 #ifdef DPD_USE_RAN_MARS
   rand_pool(0 /* unused */, lmp)
```

Comment:
@akohlmey is changing `lmp` to `_lmp` necessary? Looks like this change was missed here (for `#ifdef DPD_USE_RAN_MARS`)

Reply:
It is not a crucial change, but it reduces false positives with several code analysis tools that complain about the "lmp" parameter hiding the "lmp" class member. In 99.9% of the cases in LAMMPS it does not affect functionality, but as we are using the initializer lists more the risk to have undefined or unexpected behavior grows. So it is a change that I do whenever I notice it (it is too boring to do for the entire code base in one big change). Sorry about missing the additional entries.